akka-stream-testkit/src/main/scala/akka/stream/testkit/TestGraphStage.scala

```diff
@@ -40,22 +49,38 @@ private[testkit] class TestSinkStage[T, M](
         inHandler.onPush()
```

Comment:
I couldn't decide about push and pull and what it means there so opinions are welcome.